.github/workflows/ci.yml

```diff
@@ -58,21 +58,10 @@ jobs:
         export DEBIAN_FRONTEND=noninteractive
         apt-get update
         apt-get install -y clang-format
-    - name: check libglome format
+    - name: check format
       run: |
-        clang-format --Werror --dry-run --style=google \
-          glome.c glome.h glome_test.c
-    - name: check glome-cli format
-      run: |
-        clang-format --Werror --dry-run --style=google \
-          cli/*.c cli/*.h
-    - name: check glome-login format
-      run: |
-        clang-format --Werror --dry-run --style=google \
-          login/config.c login/crypto.c login/crypto_test.c login/lockdown.c \
-          login/login.c login/login_test.c login/main.c login/ui.c \
-          login/base64.h login/config.h login/crypto.h login/lockdown.h \
-          login/login.h login/ui.h
+        find login -name "*.c" -or -name "*.h" | \
```

Comment:
It seems this would only check login directory now?

Reply:
Thanks, fixed it.